ajax/libs/videojs-vast-plugin/package.json

```diff
@@ -0,0 +1,40 @@
+{
+  "name": "videojs-vast-plugin",
+  "version": "0.2.1",
+  "description": "A VAST plugin for VideoJS",
+  "filename": "videojs.vast.min.js",
+  "homepage": "https://theonion.github.io/videojs-vast-plugin/",
+  "repository": {
+    "type": "git",
+    "url": "git://github.com/theonion/videojs-vast-plugin.git"
+  },
+  "keywords": [
+    "video",
+    "videojs",
+    "vast"
+  ],
+  "license": "MIT",
+  "autoupdate": {
+    "source": "git",
+    "target": "git://github.com/theonion/videojs-vast-plugin.git",
+    "fileMap": [
+      {
+        "basePath": "",
+        "files": [
+          "videojs.vast.*"
+        ]
+      },
+      {
+        "basePath": "lib",
+        "files": [
+          "vast-client.js"
```

Comment:
Let's use `vast-client*` to fetch minified file and map file in the future. Thanks.